src/ui-components/empty-state/empty-state.component.tsx

```diff
@@ -15,7 +16,14 @@ export default function EmptyState(props: EmptyStateProps) {
         style={props.styles}
         className={`omrs-medium ${styles.emptyStateText}`}
       >
-        <p className="omrs-type-body-regular">{props.displayText}</p>
+        <p className="omrs-type-body-regular">
+          <Trans
+            i18nKey="emptyStateText"
+            values={{ displayText: props.displayText }}
+          >
+            This patient has no {props.displayText} recorded in the system.
```

Comment:
@FlorianRappl, I'd thought of modifying this line so that each component that uses it would provide a translated string as the `displayText` prop and this would just be put in this string which will be handled using the Trans component. Is this sound, do you reckon?

Reply:
Yes, that would be great!